src/DynamoCore/Extensions/LinterExtensionBase.cs

```diff
@@ -175,8 +204,9 @@ public virtual void Startup(StartupParams sp)
         ///<inheritdoc/>
         public virtual void Dispose()
         {
-            ReadyParamsRef.CurrentWorkspaceChanged -= OnCurrentWorkspaceChanged;
-            UnsubscribeGraphEvents(currentWorkspace);
+            //TODO - move to ShutDown once the coresponding PR is merged
```

Comment:
we should revisit this TODO after we merge https://github.com/DynamoDS/Dynamo/pull/11617